pypeit/specobj.py

```diff
@@ -60,10 +60,10 @@ class SpecObj(datamodel.DataContainer):
                  'FWHM': dict(otype=float, descr='Spatial FWHM of the object (pixels)'),
                  'FWHMFIT': dict(otype=np.ndarray,
                                  descr='Spatial FWHM across the detector (pixels)'),
-                 'THRESHOLD': dict(otype=float,
-                                  descr='Threshold used for object finding'),
                  'smash_peakflux': dict(otype=float,
-                                   descr='Peak value of the spectrum spatial profile'),
+                                        descr='Peak value of the spectral direction collapsed spatial profile'),
```

Comment:
Since you are making these changes, you should change the version number in line 53

Reply:
Good catch.